### PR TITLE
test(tile): updated calcite tile and tile-select demo into one

### PR DESCRIPTION
**Pull request by ManahariDahal**
**Related Issue:** #2641 

## Summary
- combined `tile` and `tile-select` into one demo

### CODE REVIEW

**Review by eriklharper — CHANGES_REQUESTED** [review]
Since `calcite-tile-select` is the primary component and `calcite-tile` is really just an internal component, the demo should be titled `src/demos/calcite-tile-select.html`.  You should also update the html title to: `<title>Calcite Tile Select</title>`

**Review by bstifle — APPROVED** [review]
noice!

**Review by julio8a — APPROVED** [review]
Nice work

**Review by eriklharper — CHANGES_REQUESTED** [review]
Just one more change and this should be good to go:  remove the link to the calcite-tile demo page here: https://github.com/Esri/calcite-components/blob/1cd966b9561bc8f76966146e8efef84dbe748908/src/index.html#L166-L168

**Comment by ManahariDahal** [comment]
@eriklharper thanks for the feedback. Let me know if this demo page looks good.

File: src/index.html [hunk 1]
```diff
@@ -166,9 +166,6 @@ <h2>Calcite Component Examples - WIP !</h2>
         <li>
           <calcite-link href="/demos/calcite-tabs.html">calcite-tabs</calcite-link>
         </li>
-        <li>
-          <calcite-link href="/demos/calcite-tile.html">calcite-tile</calcite-link>
```
Comment (eriklharper) [inline src/index.html:170]: Needs to be `calcite-tile-select` not `calcite-tile`.

Comment (ManahariDahal) [reply src/index.html:170]: yes, I removed that.

Comment (eriklharper) [reply src/index.html:170]: Ah, yes I see that now, sorry for the confusion!  I was forgetting this was a removal.

**Review by eriklharper — APPROVED** [review]

**Review by eriklharper — APPROVED** [review]

**Pull request merged**